<launch>

<include file="$(find openni_launch)/launch/openni.launch"/>

<node name="kinect_grabber_node" pkg="kinect_grabber" type="kinect_grabber_node" output="screen" />

<node name="pose_estimation_node" pkg="pose_estimation_uibk" type="pose_estimation_node" output="screen" />

</launch>
                                                                                                                      

